src/data/faq_de.json

```diff
@@ -570,6 +570,44 @@
+                            "Während in der ersten Version 1.0 der Schnittstelle Durchschnittswerte der Dämpfung für alle Begegnungen mit einem anderen Gerät im Laufe des Tages genutzt wurden, erfolgt die Betrachtung in der neuen Version 2.0 nun jeweils für 30-minütige Zeitfenster.",
+                            "Jedes dieser Fenster muss bestimmte Kriterien erfüllen, um berücksichtigt zu werden:",
+                            "<ul><li>Mindestens 10 Minuten innerhalb des 30-minütigen Fensters muss die Signaldämpfung unter 73dB gelegen haben</li>",
+                            "<li>Das Transmission Risk Level muss mindestens III (3) betragen. Es wird beim Teilen der Diagnoseschlüssel pro Tag gemäß definierter Kriterien wie Symptomstatus und/oder Symptombeginn festgelegt</li></ul>",
+                            "Sind diese Kriterien <b>nicht</b> erfüllt, erfolgt <b>keine</b> Zählung innerhalb der App. Es handelt sich um eine „Nicht-Risiko-Begegnung“. Im Kontaktprotokoll des Betriebssystems hingegen erfolgt trotzdem eine Zählung. Dabei handelt es sich aber nicht um einen Fehler.",
+                            "Sind aber die Bedingungen erfüllt, erfolgt eine Gewichtung der Kontaktzeiten, abhängig von der Signaldämpung:",
+                            "<ul><li>Zeiten mit einer Dämpfung <55dB werden zu 100 Prozent berechnet</li>",
+                            "<li>Zeiten mit einer Dämpfung >= 55dB und <63dB werden zu 50 Prozent berechnet</li></ul>",
+                            "Basierend auf dem Transmission Risk Level (III bis VII) wird nur ein Faktor zwischen 0,6 und 1,6 ermittelt. Mit diesem Faktor wird die gewichtete Zeit aus dem vorherigen Schritt multipliziert, woraus sich dann die gewichtete Kontaktzeit ergibt. Anhand dieser gewichteten Zeit wird nun entschieden, um was für eine Begegnung es sich bei diesem Zeitfenster handelt:",
+                            "<ul><li>Gewichtete Kontaktzeit <15 Minuten: Begegnung mit niedrigem Risiko (grün)</li>",
+                            "<li>Gewichtete Kontaktzeit >=15 Minuten: Begegnung mit erhöhtem Risiko (rot)</li></ul>",
+                            "Abschließend werden alle Zeitfenster eines Tages betrachtet (die gewichteten Kontaktzeiten) summiert. Ist deren Summe >= 15 Minuten, wird der Gesamtstatus rot, ansonsten ist dieser grün. Letztlich bedeutet das: Mehrere „grüne“ Begegnungen (Zeitfenster) können in Summe zu einem roten Status führen. Wer noch tiefer in die Materie einsteigen möchte: ",
+                            "Es stehen die aktualisierten Diagramme auf Github: <a href='https://github.com/corona-warn-app/cwa-documentation/tree/master/images/risk_calculation'>https://github.com/corona-warn-app/cwa-documentation/tree/master/images/risk_calculation</a>",
```

Comment:
Could you update this sentence how it has been done in #700?

Reply:
What do you mean - which change?

Reply:
@Ein-Tim because of the timing i need to merge this first - can you make a second PR please? Thank you!

Reply:
```suggestion
                            "Die aktualisierten Diagramme stehen auf Github: <a href='https://github.com/corona-warn-app/cwa-documentation/tree/master/images/risk_calculation'>https://github.com/corona-warn-app/cwa-documentation/tree/master/images/risk_calculation</a> zur Verfügung",
```